gcloud/credentials.py

```diff
@@ -207,6 +217,9 @@ def _get_signature_bytes(credentials, string_to_sign):
         pkey = _get_pem_key(credentials)
         if not isinstance(string_to_sign, six.binary_type):
             string_to_sign = string_to_sign.encode('utf-8')
+        if crypto is None:
+            raise EnvironmentError('pyOpenSSL must be installed to sign '
```

Comment:
Right, in general I dislike hanging indents. It's up to you- let's just be consistent within the file on whether we use them or not.